RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -2234,7 +2234,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "scripts/swiftlint.sh\n";
+			shellScript = "# Run swiftlint but filter out \"Linting ...\" to clean up output\nscripts/swiftlint.sh 2>&1 | grep -v 'Linting '\n";
```

Comment:
looks good! although maybe we can do it at the swiftlint.sh level so it's easier to find an edit in the future?

Reply:
I thought that maybe we wouldn't want to hide what that's doing when ran directly, only to clean up the output in Xcode. What do you think?

Reply:
That's why I put it here, but let me know if you think it's worth moving it there.

Reply:
Yeah, that's a good point. We could make it an optional parameter too and pass it in here.

I may be over complicating things, no need to listen to me. 

When running locally, though, I just run `swiftlint`

Reply:
That's a good point, this doesn't really add much on top of `SwiftLint`, so no reason not to put this code in there instead. I'll change it.